UnityProjects/MRTKDevTemplate/Assets/Scripts/SampleSceneHandMenu.cs

```diff
@@ -54,7 +54,11 @@ private void Awake()
             previousSceneButton.enabled = IsSceneValid(SceneManager.GetActiveScene().buildIndex - 1);
             nextSceneButton.enabled = IsSceneValid(SceneManager.GetActiveScene().buildIndex + 1);
 
+#if UNITY_2022_1_OR_NEWER
```

Comment:
Is this method not also `UNITY_2021_3_18_OR_NEWER`? The Unity docs make me believe it might be

https://docs.unity3d.com/2021.3/Documentation/ScriptReference/Object.FindFirstObjectByType.html

Reply:
I didn't see a way to add a version defines for the sample scripts and this seemed to be the next best thing... Adding the static class to core should resolve this. I'm curious is there way to add version defines for the samples?